### DISCUSSION TITLE
Is it possible to figure out what solc versions (long format) forge installs?

### DISCUSSION THREAD

**Discussion by Saw-mon-and-Natalie** [foundry-rs/foundry#1204] [Q&A] [answered]
When running `forge build`, is the `solc` version present in the contract is not installed, `forge` installs that version before compiling the contract and only outputs the version in `X.Y.Z` format. 

Is it possible to find out the long version tag for the `solc`? Also where/ in which directory do the `solc` binaries get installed?

For example: `v0.8.13+commit.abaa5c0e`

**Comment by onbjerg** [comment, marked as answer]
The binaries are put in `$HOME/.svm`, and I think the only way currently is to run the solc binary manually to get the long version :slightly_smiling_face:

**Comment by Saw-mon-and-Natalie** [comment]
Cheers, thank you. On my Windows machine using `Git Bash` I get:

```bash
$ ~/.svm/0.8.13/solc-0.8.13 --version
solc, the solidity compiler commandline interface
Version: 0.8.13+commit.abaa5c0e.Windows.msvc
```

I would be nice if `forge` had a flag to include the long version in its outputs.

**Reply by onbjerg** [reply]
Would you mind opening up an issue? It should be a relatively simple one or two liner here https://github.com/gakonst/foundry/blob/721093dc58c0d43b20c11acff04e21b34a6e76a0/cli/src/term.rs#L222-L228 :)
